islandora.module

```diff
@@ -2170,3 +2170,31 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $o = islandora_object_load($cmodel);
+      if (is_object($o)) {
+        $rows[] = array(l($o->label, "islandora/object/{$o->id}"));
+      }
+      else {
+        $rows[] = array(t("@cmodel  - (This content model is not in this Islandora repository.)", array('@cmodel' => $cmodel)));
+      }
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
+  $table = array(
+    '#theme' => 'table',
+    '#header' => array($header),
+    '#rows' => $rows,
+    '#empty' => t('No Islandora content model associated with this object.'),
+  );
+  $rendered_table = drupal_render($table);
```

Comment:
Blanking the default display is expected, as it is how it is implemented: [The default is only added if there's no other implementations](https://github.com/Islandora/islandora/blob/7.x/islandora.module#L931-L934)... On further thought, the implementation here should probably go into [the default implementation](https://github.com/Islandora/islandora/blob/7.x/islandora.module#L953), instead of being separate... Would result in the default never being displayed.